tests/TestData/SimModel4_ExampleInput04.xml

```diff
@@ -118,13 +118,11 @@
 	<VariableList>
 		<V id="189259446" name="Educt" path="TopContainer" unit="L" initialValueFormulaId="415719451" entityId="189259446">
 			<ScaleFactor>1</ScaleFactor>
-			<RHSFormulaList>
-			</RHSFormulaList>
+			<RHSFormulaList/>
 		</V>
```

Comment:
seems to be a bug in libXml2 under linux. 
If the xml list is given in the form `<MyList></MyList>`, libxml assumes there are subnodes and crashes. `<MyList/>` is fine.